opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -228,25 +228,28 @@ message Metric {
 // AggregationTemporality is not included. Consequently, this also means
 // "StartTimeUnixNano" is ignored for all data points.
 message Gauge {
+  // The time series point values for this gauge.
```

Comment:
It may be data points all with the same timestamp and different attributes and calling that "time series" I think may be confusing.

Perhaps just say
> // The data points for this gauge.

Although I am not sure there is much value in such comments. Are comments mandated? Should we skip low-value comments like this?

Reply:
Comments are mandated now, which is why i added.

This structure actually HAS caused some confusion in the past.  The wording I use is to match our datamodel.  I think what's confusing you is that "time series" is both singular and plural, and I'm using the plural here.  I actually WANTED to have this comment denote that you'll get multiple time series in here (where time series = unique label sets) for a given metric, even if they all have the same timestamp.

SO I think we both agree there's something important to call out here.  Not sure how to phrase it.